src/components/Sum.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+function Sum(obj) {
```

Comment:
сделай сразу деструктуризацию, чтоб не писать obj.a, obj.b 
```suggestion
function Sum({ a, b }) {
```

Reply:
+